schema.sql

```diff
@@ -1,5 +1,13 @@
 /* Database schema to keep the structure of entire database. */
 
+CREATE DATABASE vet_clinic;
+
+
 CREATE TABLE animals (
-    name varchar(100)
+    id INT GENERATED ALWAYS AS IDENTITY,
+    name varchar,
```

Comment:
- [ ] When giving the name type of varchar you should also give its size. Example:
`name varchar (255)`